doc/ref/states/requisites.rst

```diff
@@ -332,6 +377,50 @@ to Salt ensuring that the service is running.
         - name: /etc/ntp.conf
         - source: salt://ntp/files/ntp.conf
 
+watch_any
+~~~~~~~~~
+
+.. versionadded:: Oxygen
+
+The state containing the ``watch_any`` requisite is defined as the watching
+state. The states specified in the ``watch_any`` statement are defined as the watched
+states. When the watched states execute, they will return a dictionary containing
+a key named "changes".
+
+If the "result" of any of the watched states is ``True``, the watching state *will
+execute normally*, and if all of them are ``False``, the watching state will never run.
+This part of ``watch`` mirrors the functionality of the ``require`` requisite.
+
+If the "result" of any of the watched states is ``True`` *and* the "changes"
+key contains a populated dictionary (changes occurred in the watched state),
+then the ``watch`` requisite can add additional behavior. This additional
+behavior is defined by the ``mod_watch`` function within the watching state
+module. If the ``mod_watch`` function exists in the watching state module, it
+will be called *in addition to* the normal watching state. The return data
+from the ``mod_watch`` function is what will be returned to the master in this
+case; the return data from the main watching function is discarded.
+
+If the "changes" key contains an empty dictionary, the ``watch`` requisite acts
+exactly like the ``require`` requisite (the watching state will execute if
+"result" is ``True``, and fail if "result" is ``False`` in the watched state).
+
+.. code-block:: yaml
+
+    apache2:
+      service.running:
+        - watch_any:
+          - file: /etc/apache2/sites-available/site1.conf
+          - file: /etc/apache2/sites-available/site2.conf
+      file.managed:
+        - name: /etc/apache2/sites-available/site1.conf
+        - source: salt://apache2/files/site1.conf
+      file.managed:
+        - name: /etc/apache2/sites-available/site2.conf
+        - source: salt://apache2/files/site2.conf
+
+In this example, the service will be reloaded/restarted if the either of the state
```

Comment:
I think this statement could be a bit more clear/specific. How about "In this example, the service will be reloaded/restarted if either of the ``file.managed`` states has a result of ``True`` and has changes."?